change/@itwin-one-click-lca-react-e728c345-afb6-40e3-91c6-655b69d22217.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Changes in One Click LCA widget to support dev / qa environment",
```

Comment:
3rd party consumers don't really care about DEV/QA environment - they can only use our PROD environment. This PR should be adding a way to provide a custom client implementation, not just changing the urls.